server/src/shared/utils/getTemplate.ts

```diff
@@ -0,0 +1,13 @@
+import { template } from 'dot';
+
+export const getTemplatedString = (data: any, file: string): string => {
```

Comment:
I do not like the `any` over here, you can specifically give the right type here